### ISSUE TITLE
Consider support for "link click"+keyboard shortcut within content

### ISSUE THREAD

**Issue by jonathanKingston** [mozilla/testpilot-containers#518]
To improve the use of containers should we consider a shortcut like Alt+click for opening a container in "Default"?

We could also make Alt+click and drag down/long press work like the back/forward buttons in Firefox to open a container list.

/cc @groovecoder for discussion

**Comment by jonathanKingston** [comment]
Alt+Ctrl/Alt+Shift might be needed at least on Ubuntu as they use Alt+Click for window dragging.
I can't even seem to override this within Firefox itself.

**Comment by jonathanKingston** [comment]
The ugly looking PR I added is a proof of concept that looks a little like this:

![selection_672](https://cloud.githubusercontent.com/assets/338988/26728204/58f9847a-47a1-11e7-909e-3696c50d40f7.png)

**Comment by jonathanKingston** [comment]
This could also be expanded to using focused links for Alt+tab too. I select the first item by default so keyboard users can tab here.

**Comment by bakulf** [comment]
This is nice. But it's similar to right click -> open link in container X.

**Comment by jonathanKingston** [comment]
@bakulf yeah its very similar. I find that menu too hard to use though. Allowing it for keyboard navigation would be a significant advantage too.
Users are familiar with Ctrl+Clicking a lot right click, move down two, across requires a lot more overhead for a common action.

**Comment by jonathanKingston** [comment]
![selection_677](https://user-images.githubusercontent.com/338988/26857785-1b019d06-4b25-11e7-96ae-c9b2df5a6715.png)
![selection_678](https://user-images.githubusercontent.com/338988/26857787-1e433c0e-4b25-11e7-8b49-7458f8a84a73.png)

Added drop shadow and made it much like #433. If you get time @groovecoder this is now a working demo would be cool to get your thoughts. It's still not super bug free etc however I think it demos how usable this can be over the right click context menu.

**Comment by jonathanKingston** [comment]
Outstanding issues:

- [ ] Some events don't seem to be cancelled, on mozilla.org the side menu still opens
- [ ] Some urls break and instead open a blank tab
- [ ] Keyboard navigation doesn't seem to work
  - [ ] Tabbing out of the menu what should happen
  - [ ] Arrow keys etc
- [ ] Probably needs a quick subtle animation
- [ ] Prevent text highlight
- [ ] Font sometimes is overridden

**Comment by dannycolin** [comment]
That option would be great but It would be nice to consider that some people use the caret mode (F7). Does the list would be accessible to them.
